ansible_collections/arista/avd/plugins/plugin_utils/eos_validate_state_utils/avdtestbase.py

```diff
@@ -82,6 +114,23 @@ def log_skip_message(
         log_level = logging.getLevelName(logging_level.upper())
         LOGGER.log(log_level, log_msg)
 
+    def is_interface_shutdown(self, interface: dict, host: str | None = None) -> bool:
+        """
+        Check if an interface is shutdown or not, considering EOS defaults.
+
+        Args:
+            interface (str): The interface to verify.
+            host (str): Host to verify. Defaults to the host running the test.
+
+        Returns:
+            bool: Returns the interface shutdown key's value, or `interface_defaults.ethernet` if not available.
```

Comment:
so is the default only valid for ethernet interfaces then and, for instance, does not impact loopbacks and so on?

Reply:
That's correct. Good catch I fixed the logic a bit in 22836aff3e4102ab512a54b400ec10a4213cff27 to reflect that.